lib/ddtrace/contrib/graphql/patcher.rb

```diff
@@ -31,7 +31,7 @@ def patch_schema!(schema)
             schema.use(
               ::GraphQL::Tracing::DataDogTracing,
               # By default, Tracing::DataDogTracing indirectly delegates the tracer instance
-              # to +Datadog.tracer+. If we provide a tracer argument here it will be eagerly cached,
+              # to +Datadog::Tracing.tracer+. If we provide a tracer argument here it will be eagerly cached,
               # and Tracing::DataDogTracing will send traces to a stale tracer instance.
               service: service_name,
               analytics_enabled: analytics_enabled,
```

Comment:
‼️ Actually, in master `GraphQL::Tracing::DataDogTracing` still delegates to `Datadog.tracer` => https://github.com/rmosolgo/graphql-ruby/blob/master/lib/graphql/tracing/data_dog_tracing.rb#L51 .

I think we need to get in contact with upstream and ask them to add support for 1.0.0.

Reply:
Yes, this is true, and our tests verified this. We haven't released 1.0 with this API change, so there's no way for them to build against it. We should submit a PR to fix it after 1.0 is released.

Reply:
Since they'll probably need to support both `Datadog.tracer` as well as `Datadog::Tracing.tracer`, perhaps we could already report it to them?

(I would suggest not changing the comment above until it's actually true -- or maybe deleting the comment, as it can be misleading)